test/parallel/test-whatwg-url-searchparams-getall.js

```diff
@@ -12,7 +12,7 @@ const { test, assert_equals, assert_true, assert_array_equals } =
    License: http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html
 */
 /* eslint-disable */
-test(function() {
+test(() => {
```

Comment:
As per the comment above, it seems we should not change this fragment.

Reply:
> Modifications to them should be upstreamed first.

I'm sorry to have missed it.
I'll revert changes in this file.